mlflow/gateway/constants.py

```diff
@@ -2,3 +2,7 @@
 MLFLOW_GATEWAY_CRUD_ROUTE_BASE = "/api/2.0/gateway/routes/"
 MLFLOW_GATEWAY_ROUTE_BASE = "/gateway/"
 MLFLOW_QUERY_SUFFIX = "/invocations"
+
+# Provider constants
+MLFLOW_AI_GATEWAY_ANTHROPIC_MAX_TOKENS = 1000000
+MLFLOW_AI_GATEWAY_ANTHROPIC_DEFAULT_TOKENS = 200000
```

Comment:
Instead of using environment variables, can we introduce a provider-specific request model and implement validation logic there?

Reply:
We currently use a single request model for all providers for each route type, but we could extend it to support provider-specific restrictions. I'm not sure if this is really possible though. The advantage is you can see the restrictions on the interactive API documentation.

Reply:
Never mind this :)